ui/tests/unit/queries.spec.js

```diff
@@ -24,6 +24,16 @@ describe("App", () => {
             }
           ],
           __typename: "IdentityPaginatedType"
+        },
```

Comment:
I think we should include a test to evaluate the parameters we are passing to the method are correctly set.